python/ray/autoscaler/kubernetes/example-full.yaml

```diff
@@ -139,6 +139,12 @@ available_node_types:
               # cause problems for other pods.
               memory: 512Mi
   head_node:
```

Comment:
same as previous comment. on max_workers

Reply:
I think max_workers should stay in the example-fulls, because example-full yamls are supposed to document everything.

They're not in the defaults yamls to avoid improperly overriding things.